test/integration/router/router_http_server.go

```diff
@@ -19,6 +19,8 @@ import (
 	kapi "k8s.io/kubernetes/pkg/api"
 
 	"github.com/openshift/origin/pkg/cmd/util"
+
+	_ "github.com/openshift/origin/pkg/api/install"
```

Comment:
how is this related to the PR?

Reply:
> how is this related to the PR?

if the APIs aren't installed, their fake apiserver fails.